openapi.yaml

```diff
@@ -351,6 +481,26 @@ components:
       pattern: 'A[1-9]([0-9]{1,3})?(\/A[1-9]{1,3})?'
       example: "A1"
 
+    RoadsAffected:
+      type: object
+      properties:
+        roadCounter:
+          type: array
+          items:
+            $ref: '#/components/schemas/RoadCounter'
+
+    RoadCounter:
+      type: object
+      example: {
+        "road": "A1",
+        "counter": "99"
+      }
+      properties:
+        road:
+          $ref: '#/components/schemas/RoadId'
+        counter:
+          type: number
```

Comment:
Sorry, just run the openapi check again locally, and this either needs to be a `string' not a `number' or line 496 needs to be changed.

```
 496:20    error  oas3-valid-schema-example  "counter" property type must be number       components.schemas.RoadCounter.example.counter
``` 

You can run the spectral lint by yourself if you have docker installed with 
`docker run --rm -it -v $(pwd):/tmp stoplight/spectral lint --ruleset "/tmp/.spectral.yaml" "/tmp/openapi.yaml"`